frontend/src/pages/UserPage.jsx

```diff
@@ -1,25 +1,26 @@
-import { NavLink, Outlet } from "react-router-dom";
+import { NavLink, Outlet, useOutletContext } from "react-router-dom";
 
 export default function UserPage() {
+  const { auth } = useOutletContext();
   const navUser = [
     {
       id: 1,
-      path: "/user/info",
+      path: `/user/info/${auth.id}`,
```

Comment:
This is a security issue, you already have all needed informations to retrieve the current user in your database.